v21.2/restoring-backups-across-versions.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Restoring Backups Across Versions
+summary: CockroachDB's support policy for restoring backups across versions.
+toc: true
+docs_area: manage
+---
+
+This page describes the support Cockroach Labs provides for restoring backups across major versions of CockroachDB. The sections outline the support and examples for the most common use cases when restoring backups across versions:
+
+- [Support for restoring backups into a newer version](#support-for-restoring-backups-into-a-newer-version)
+- [Restore into a newer version](#restore-into-a-newer-version)
+- [Support for long-term backup archival](#support-for-long-term-backup-archival)
+
+## Support for restoring backups into a newer version
+
+Cockroach Labs supports restoring a backup on one major version following the major version on which you took the backup. Therefore, if you took a backup on major version N, you can restore it on major version N+1.
```

Comment:
This is hard to express, the current text sounds a bit confusing to me. How does this sound?

You can restore a backup taken on a cluster on a specific major version (e.g. version N), into a cluster that is on the next major version (e.g. version N+1).

Reply:
I've altered this a little to match up here.